index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>A Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header id="header">
+    <img id="header-img" src="eye.png" alt="">
+    <nav id="nav-bar">
+    <a class="nav-link" href="#mail">Home</a>
+    <a class="nav-link" href="#text-container">About</a>
+    <a class="nav-link" href="#video-section">Media</a>
+    </nav>
+    </header>
+    <section id="mail">
+    <h2 class="heading">Handcrafted, home-made masterpieces</h2>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input type="email" id="email" placeholder="enter email" name="email"> <br> <br>
+      <input type="submit" id="submit" value="subscribe">
+    </form>
+  </section>
+    <section id="text-container">
+      <div id="info">
+        <h3>traffic barricade</h3>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+      </div>
+      <div id="info">
+        <h3>traffic barricade</h3>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+      </div>
+      <div id="info">
+        <h3>traffic barricade</h3>
```

Comment:
I think you should add padding to separate the title text from the body text to the make the work look professional.

I suggest that if you could add colors to the title text it will make your work more attractive.